2024/calcite-react/demo/src/App.scss

```diff
@@ -0,0 +1,8 @@
+header {
```

Comment:
oh, I didn't realize that vite has scss support out of the box - that's cool

(optional) though we could still use regular .css here to make the demo app more approachable for people not familiar with scss

Reply:
i installed sass but i'm fine either way. removed sass and converted style files to .css